lambda/downsample_volume.py

```diff
@@ -147,6 +147,57 @@ def exists(self, key):
 
 #### Main lambda logic ####
 
+def _conv(a, kernel):
```

Comment:
_conv doesn't look to be called in this code.
and there is a conv on line 198.  should this be changed to conv?

Reply:
oh shoot — yeah definitely should be `conv` (or the other should be `_conv`) 😬